README.md

```diff
@@ -5,4 +5,4 @@ In the CONTRIBUTORS file simply add your name, institution, country in the follo
 `| Serial No. |[Your_Name](https://github.com/<your_github_username>) | <Your Institution Name> | <Your Country Name> |`  
 Note that `Your Institution Name` and `Your Country Name` are optional and you may leave them blank if you wish.
 
-Note that your commit message should include `Fixes issue https://github.com/ayan-b/hacktoberfest-firstPR/issues/1`.
+Note that your commit message should include `Fixes issue https://github.com/ayan-b/hacktoberfest-firstPR/issues/1`.
```

Comment:
Sorry, but what have you changed here?

Reply:
No problem.

Reply:
Sorry, but it is still showing. You can move the HEAD before this commit and then add the changes and commit again.